packages/cardano-graphql-provider/src/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../dist/",
+    "rootDir": "."
+  }
+}
```

Comment:
You must declare the dependency on the core package using [project references](https://www.typescriptlang.org/docs/handbook/project-references.html). Have a look at the `golden-test-generator` tsconfig for reference:

https://github.com/input-output-hk/cardano-js-sdk/blob/1a9f2ff3340defda99f392c0eca153f854f57311/packages/golden-test-generator/src/tsconfig.json#L7-L9